python-code/set-sample/sample2.py

```diff
@@ -1,3 +1,12 @@
-import os
-value = 10
-print("Hello code world")
+import math
```

Comment:
This is a  Unused import math
```suggestion 
```

 from our GitHub Enhacement Bot